### ISSUE TITLE
Explore the implementation of hateos of list entry points 

### ISSUE THREAD

**Issue by ypriverol** [HUPO-PSI/proxi-schemas#10]
Would be great if we can implements hateos for every entry point that list collections.

**Comment by jjcarver** [comment]
I agree and this is fairly simple to do. However, the only link types I can think of would be "self" (a link to the full individual record for each list item), and maybe links to the previous page and/or next page, if present. Are there any other link types we should consider including?
